scripts/provisioning/resetcmd.py

```diff
@@ -28,6 +28,8 @@
 from s3backgrounddelete.cortx_s3_constants import MESSAGE_BUS
 from setupcmd import SetupCmd
 
+services_list = ["haproxy", "s3backgroundconsumer", "s3backgroundproducer", "slapd", "s3authserver", "s3server@*"]
```

Comment:
you can have s3server@* after s3backgroundproducer and also slapd after s3authserver -- Basically shutting producer before consumer

Reply:
done